Cutelyst/application.cpp

```diff
@@ -441,6 +442,96 @@ bool Application::enginePostFork()
     return true;
 }
 
+void Application::addTranslator(const QLocale &locale, QTranslator *translator)
+{
+    Q_D(Application);
+    Q_ASSERT_X(translator, "add translator to application", "invalid QTranslator object");
+    if (d->translators.contains(locale)) {
```

Comment:
It's usually better to check if it contains with an iterator, that way if  there is a match you already have a valid iterator pointing to the data.